src/lib.rs

```diff
@@ -22,16 +22,17 @@
 //! * Windows
 //! * illumos
 //! * Android
+//! * iOS
```

Comment:
テスト回したいので無をcommitさせてもらいます 🙇‍♂️ 
```suggestion
//! * iOS 
```